src/Crypto/AesGcmDecryptingStream.php

```diff
@@ -83,6 +83,7 @@ public function createStream()
                 $this->keySize
             ));
         } else {
+            // phpcs:ignore PHPCompatibility.FunctionUse.NewFunctionParameters -- False positive, never used in php <= 7.0
```

Comment:
In my loosely held opinion in the tests is best, as then your rules and your tests are right next to each other in your editor view.

Reply:
I agree that using the comments is rather ugly but it's the only way to ignore specific sections of code.

Maybe we could go for a middleground where we have multiple configuration-files:
* All rules, all files except the ones with exceptions
* All rules except false positive rule 1, single file which had the false positive
* All rules except false positive rule 2, single file which had the false positive
* ...

This way, we would ignore the false positive in a whole file, which might lead to a false negative in case a rule would trigger twice in a file with one false positive.

This would allow to have the bulk of the files linted with all rules and files with false positives would only have one rule ignored.